lint-configs/.clang-format

```
@@ -56,6 +57,7 @@ BreakBeforeBraces: "Custom"
 BreakBeforeConceptDeclarations: "Always"
 BreakBeforeInlineASMColon: "OnlyMultiline"
 BreakBeforeTernaryOperators: true
+BreakBinaryOperations: "RespectPrecedence"
```

```suggestion
BreakBinaryOperations: "Never"
```

I prefer this since RespectPrecedence may look odd and regardless, we shouldn't be writing long binary expressions where precedence rules come into play without parentheses.

---

kk, makes sense